admin-dev/themes/new-theme/js/app/cldr/number-formatter.js

```diff
@@ -1,12 +1,12 @@
 /**
- * 2007-2019 PrestaShop.
+ * 2007-2019 PrestaShop SA and Contributors
  *
  * NOTICE OF LICENSE
  *
- * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * This source file is subject to the Open Software License (OSL 3.0)
```

Comment:
@matks Do you really want to change the license ?

Reply:
Yes it was a mistake, AFL is for modules while the main project is OSL